lib/restate-constructs/service-deployer.ts

```diff
@@ -91,48 +96,44 @@ export class ServiceDeployer extends Construct {
     id: string,
     /**
      * Allows the custom resource event handler properties to be overridden. The main use case for this is specifying
-     * VPC and security group settings for Restate environments that require it.
+     * VPC and security group settings for Restate environments that require it. The event handler must be able
+     * to reach the S3 API, so if the subnet has no egress, it will need an S3 VPC endpoint.
      */
-    props?: Pick<
```

Comment:
This will probably break existing stacks - IIRC Ax Semantics might have been using `depsLockFilePath`. Easy fix, just drop the attribute, as it was a workaround for some `esbuild` weirdness, but maybe worth mentioning in release notes - or even making the next release a major version bump.